src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/Common/COMWrappersImpl.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests.Common
+{
+    public class ComWrappersImpl : ComWrappers
+    {
+        public const string IID_TestQueryInterface = "1F906666-B388-4729-B78C-826BC5FD4245";
```

Comment:
It might be worth adding a comment here to remind us that this IID is for a made up interface for these tests and not a well-known documented interface.

Reply:
Very reasonable. Will do.

Reply:
```suggestion
        // Doesn't represent a real interface. The value is only used to support a call to QueryInterface for testing.
        public const string IID_TestQueryInterface = "1F906666-B388-4729-B78C-826BC5FD4245";
```